package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@skiff-org/typed-envelopes",
-  "version": "0.1.0",
+  "version": "0.1.1",
   "license": "AGPL-3.0",
   "author": "skiff-org",
   "main": "./dist/index.js",
```

Comment:
does this still need the `./`?

Reply:
I don't see the harm in keeping this the same. I don't think it's the same as the `gitignore`-style includes later on in this file. I've imported this package into a sandbox TS project and the import paths looked sane, and that was really the driving force behind ctix + this packaging stuff.


<img width="516" alt="Screen Shot 2021-04-20 at 1 41 20 PM" src="https://user-images.githubusercontent.com/947772/115461298-1dd14e00-a1de-11eb-9b25-4fd52b3817c5.png">